utils/RSF/onReciverRecon.ts

```diff
@@ -0,0 +1,37 @@
+import constants, { RSFapiSequence } from '../../constants/index'
+import _ from 'lodash'
+import { isObjectEmpty } from '../../utils/index'
+import { validateSchema } from '../../utils/index'
+import { logger } from '../../shared/logger'
+
+const checkOnReceiverRecon = (data: any) => {
+  const issueObj: any = {}
```

Comment:
Please revise the naming here and in subsequent occurrences to align with RSF-related object nomenclature